packages/openapi-v2/src/keys.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/openapi-v2
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+// Note for review purpose, will be deleted when land the PR:
+// We can have a discussion for each key's name and its value:
+// I will add my proposal in a git comment under this line
```

Comment:
The `namespace` is called `ControllerKeys`, since they are used for processing controller class metadata.

The name of each key is changed from `REST_*` to `*` because the decorators are not tightly coupled with the `rest` package after being extracted to a single module.

And the values are renamed from `rest:*` to `controller:*`

I would like to hear more opinions on the names, thanks!

Reply:
I like decoupling `REST` from the keys here. On top of that though, I think the keys  should also have the word `spec` or something akin to it included as well since the metadata attached to those keys all are related to the specs.